docs/source/array-numpy-compatibility.rst

```diff
@@ -0,0 +1,192 @@
+Compatibility with Numpy functions
+==================================
+
+The following table describes the compatibilities between `numpy` and `dask.array`
+functions.
+Please be aware that some inconsistencies with the two versions may exist.
+
+.. |check| raw:: html
+
+    &#9989;
+
+.. |uncheck| raw:: html
+
+    <input disabled="" type="checkbox">
+
+
++--------------------------------+-----------+------------------------+----------------------------+
+| Numpy                          | from      | from                   | partial or no              |
+| function                       | numpy     | dask.array             | implementation             |
++================================+===========+========================+============================+
+| :func:`numpy.absolute`         | |check|   |     |uncheck|          |   |uncheck|                |
++--------------------------------+-----------+------------------------+----------------------------+
+| :func:`numpy.abs`              | |check|   |     |uncheck|          |   |uncheck|                |
++--------------------------------+-----------+------------------------+----------------------------+
+| :func:`numpy.add`              | |check|   |     |uncheck|          |   |uncheck|                |
++--------------------------------+-----------+------------------------+----------------------------+
+| :func:`numpy.all`              | |uncheck| |     |check|            |   |uncheck|                |
++--------------------------------+-----------+------------------------+----------------------------+
+| :func:`numpy.allclose`         | |uncheck| |     |check|            |   |uncheck|                |
++--------------------------------+-----------+------------------------+----------------------------+
+| :func:`numpy.amax`             | |uncheck| | :func:`dask.array.max` |   |uncheck|                |
++--------------------------------+-----------+------------------------+----------------------------+
+| :func:`numpy.amin`             | |uncheck| | :func:`dask.array.min` |   |uncheck|                |
```

Comment:
The fact that these are the only Dask array functions (so far) that don't match up with their NumPy equivalent is perhaps an indication that they should be renamed, especially since I could see users confusing these with [`numpy.maximum`](https://numpy.org/doc/stable/reference/generated/numpy.maximum.html).